lua/telescope/previewers/buffer_previewer.lua

```diff
@@ -78,7 +78,7 @@ previewers.file_maker = function(filepath, bufnr, opts)
             if opts.callback then opts.callback(bufnr) end
         end)})
       else
-        path.read_file_async(filepath, vim.schedule_wrap(function(data)
+        path:new(filepath):read(vim.schedule_wrap(function(data)
```

Comment:
I actually prefer `_read_async` here i know its kinda cool that it detects it automatically but we can save the `if` check here